core/server/app.js

```diff
@@ -22,12 +23,18 @@ module.exports = function setupParentApp() {
     // (X-Forwarded-Proto header will be checked, if present)
     parentApp.enable('trust proxy');
 
+    parentApp.use(responseTime());
```

Comment:
I would like to keep express formatting. See:

### with express:
```
INFO undefined [2016-10-20 12:12:33] "GET /ghost/img/ghosticon.jpg" 301 3.447ms
INFO 127.0.0.1 [2016-10-20 12:12:33] "GET /" 200 154.220ms
INFO 127.0.0.1 [2016-10-20 12:12:33] "GET /assets/css/screen.css?v=8fcd3ab6e5" 200 9.465ms
INFO 127.0.0.1 [2016-10-20 12:12:33] "GET /assets/js/jquery.fitvids.js?v=8fcd3ab6e5" 200 3.092ms
INFO 127.0.0.1 [2016-10-20 12:12:33] "GET /assets/js/index.js?v=8fcd3ab6e5" 200 2.878ms
INFO 127.0.0.1 [2016-10-20 12:12:33] "GET /assets/fonts/casper-icons.woff?v=1" 304 2.789ms
```

### with our internal solution:
```
INFO undefined [2016-10-20 12:13:07] "GET /ghost/img/ghosticon.jpg" 301 3
INFO 127.0.0.1 [2016-10-20 12:13:07] "GET /" 200 163
INFO 127.0.0.1 [2016-10-20 12:13:07] "GET /assets/css/screen.css?v=e389b8ca44" 200 8
INFO 127.0.0.1 [2016-10-20 12:13:07] "GET /assets/js/jquery.fitvids.js?v=e389b8ca44" 200 3
INFO 127.0.0.1 [2016-10-20 12:13:07] "GET /assets/js/index.js?v=e389b8ca44" 200 3
INFO 127.0.0.1 [2016-10-20 12:13:07] "GET /assets/fonts/casper-icons.woff?v=1" 304 2
```